kedro/framework/cli/starters.py

```diff
@@ -210,14 +211,17 @@ def _validate_range(start, end):
     def _validate_selection(add_ons: list[str]):
         for add_on in add_ons:
             if int(add_on) < 1 or int(add_on) > len(ADD_ONS_DICT):
-                message = f"'{add_on}' is not a valid selection.\nPlease select from the available add-ons: 1, 2, 3, 4, 5."  # nosec
+                message = f"'{add_on}' is not a valid selection.\nPlease select from the available add-ons: 1, 2, 3, 4, 5, 6."  # nosec
                 click.secho(message, fg="red", err=True)
                 sys.exit(1)
 
     if add_ons_str == "all":
         return list(ADD_ONS_DICT)
     if add_ons_str == "none":
         return []
+    # Guard clause if add_ons_str is None, which can happen if prompts.yml is removed
+    if not add_ons_str:
+        return []
```

Comment:
When `prompts.yml` is empty or doesn't exist we should default add-ons to be `"none"` hence we return an empty dictionary. We don't want `add_ons_str` to be `null`.